c-ext/decompressor.c

```diff
@@ -299,15 +299,15 @@ PyObject *Decompressor_decompress(ZstdDecompressor *self, PyObject *args,
         goto finally;
     }
 
-    decompressedSize = ZSTD_getFrameContentSize(source.buf, source.len);
-
-    if (ZSTD_CONTENTSIZE_ERROR == decompressedSize) {
+    ZSTD_frameHeader zfh;
```

Comment:
nit: maybe call it `frameHeader`

Reply:
yup, copy pasted this from somewhere around the code, but frameHeader sounds a lot better